spec/lib/pensio_api/ecommerce_spec.rb

```diff
@@ -7,22 +7,25 @@
   end
 
   describe '.create_payment_request' do
-    let(:response) { PensioAPI::Ecommerce.create_payment_request(reservation_arguments.merge({timeout: 10})) }
+    let(:response) {PensioAPI::Ecommerce.create_payment_request(reservation_arguments.merge({timeout: 10}))}
     it 'returns an instance of PensioAPI::Responses::GatewayURL' do
       expect(PensioAPI::Request).to receive(:post).with("/merchant/API/createPaymentRequest",
-        { :basic_auth=>{:username=>"test_user", :password=>"password"},
-          :headers=>{"Content-Type"=>"application/x-www-form-urlencoded; charset=utf-8"},
-          :body=>{:terminal=>"Pensio Test Terminal",
-          :shop_orderid=>"Test Payment",
-          :amount=>123.45,
-          :currency=>"eur"},
-          :timeout=>10 }).and_call_original
+                                                        {:basic_auth => {:username => "test_user", :password => "password"},
```

Comment:
Mostly a whitespace/formatting difference here, but we also check for the new version header